### PR TITLE
WIP: Change docker image name

### PR DESCRIPTION
**Pull request by vmarkovtsev**
As Francesc suggested in https://github.com/src-d/devrel/issues/1

### CODE REVIEW

File: README.md
Comment (dpordomingo) [inline README.md]: But this is the name of the image when locally created.
Is this really required by https://github.com/src-d/devrel/issues/1 ?

Comment (vmarkovtsev) [reply README.md]: Sure. I mean that our docker name is src-d and we suppose that there is no difference between a local image and the dockerhub's.

Comment (dpordomingo) [reply README.md]: fair enough

**Review by dpordomingo — APPROVED** [review]
lgtm

**Comment by bzz** [comment]
@vmarkovtsev thank you for contribution! Wonder, if that is that a cross-organizational effort, like are we changing it in all repos? Would prefer to do that all together.

Also https://hub.docker.com/u/src-d/ results in 404 to me, so I suggest we rename it only after the whole thing works. Have added WIP in the PR title, until dockerHub org is renamed.

But, please, let me know if I'm missing something here.

**Comment by vmarkovtsev** [comment]
This is just my general notice how the containers have been named de-facto, albeit srcd instead of src-d. The rest is up to @campoy @marnovo

**Comment by bzz** [comment]
Got it, thank you for explanation!
If you would not mind changing `src-d -> srcd` for now until dockerHub org renamed, we could merge this asap.

**Comment by vmarkovtsev** [comment]
IMHO it is fine to name it srcd right now because Francesc or Marcelo will rename the whole org soon.

**Review by bzz — APPROVED** [review]

**Pull request merged**
